docker-compose.yaml

```
@@ -0,0 +1,32 @@
+    volumes:
+      - ./lightspeed-stack.yaml:/app-root/lightspeed-stack.yaml:Z
+    environment:
+      - OPENAI_API_KEY=${OPENAI_API_KEY}
+    depends_on:
+      - llama-stack
+    networks:
+      - lightspeednet
+
+networks:
+  lightspeednet:
+    driver: bridge
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)